packages/autoloader/src/autoload.php

```diff
@@ -78,6 +78,7 @@ function autoloader( $class_name ) {
 					$class_name,
 					array(
 						'Automattic\Jetpack\JITM',
+						'Automattic\Jetpack\Connection\Client',
```

Comment:
we will need to add all of them, maybe we should remove the check completely? it is going to become useless.

Reply:
My comment above ^^ makes this line unnecessary